packages/components/src/components/hds/app-frame/index.ts

```diff
@@ -33,47 +33,30 @@ export interface HdsAppFrameSignature {
 }
 
 export default class HdsAppFrameComponent extends Component<HdsAppFrameSignature> {
-  /**
-   * Indicates if the "header" container should be displayed
-   *
-   * @param hasHeader
-   * @type {boolean}
-   * @default true
-   */
+  // Indicates if the "header" container should be displayed
   get hasHeader(): boolean {
     return this.args.hasHeader ?? true;
   }
 
-  /**
-   * Indicates if the "sidebar" container should be displayed
-   *
-   * @param hasSidebar
-   * @type {boolean}
-   * @default true
-   */
+  // Indicates if the "sidebar" container should be displayed
   get hasSidebar(): boolean {
     return this.args.hasSidebar ?? true;
   }
 
-  /**
-   * Indicates if the "footer" container should be displayed
-   *
-   * @param hasFooter
-   * @type {boolean}
-   * @default true
-   */
+  // Indicates if the "footer" container should be displayed
   get hasFooter(): boolean {
     return this.args.hasFooter ?? true;
   }
 
-  /**
-   * Indicates if the "modals" container should be displayed
-   *
-   * @param hasModals
-   * @type {boolean}
-   * @default true
-   */
+  // Indicates if the "modals" container should be displayed
   get hasModals(): boolean {
     return this.args.hasModals ?? true;
   }
+
+  // Get the class names to apply to the component.
+  get classNames(): string {
```

Comment:
I think at this point you can revert the changes to the getter and move the single class back to the template file